src/renderers/dom/client/syntheticEvents/SyntheticEvent.js

```diff
@@ -195,3 +200,45 @@ SyntheticEvent.augmentClass = function(Class, Interface) {
 PooledClass.addPoolingTo(SyntheticEvent, PooledClass.fourArgumentPooler);
 
 module.exports = SyntheticEvent;
+
+/*
+ * Utility functions
+ */
+
+/**
+  * Helper to nullify syntheticEvent instance properties when destructing
+  *
+  * @param {object} SyntheticEvent
+  * @param {String} propName
+  */
+function getPooledWarningPropertyDefinition(propName, getVal) {
```

Comment:
This function will be dead-code eliminated when `!__DEV__` because it's only referenced in `if (__DEV__)` statements.